packages/cli/src/utils/getQuestions.ts

```diff
@@ -0,0 +1,45 @@
+import chalk from "chalk";
+
+export interface GetQuestionsInput {
+  services: {
+    name: string;
+    value: string;
+  }[];
+}
+
+export const getQuestions = (input: GetQuestionsInput) => {
```

Comment:
Could you specify the type?
```suggestion
export const getQuestions = (input: GetQuestionsInput):  QuestionCollection[] => {
```